tests/test_setting_env.cpp

```diff
@@ -439,4 +439,6 @@ TEST(test_layer_setting_env, vkuGetLayerSettingValues_String) {
     EXPECT_EQ(2, value_count);
 
     vkuDestroyLayerSettingSet(layerSettingSet, nullptr);
+
+    SetEnv("VK_LUNARG_TEST_MY_SETTING=");
```

Comment:
FFTI

This may be a case where a class for environment variables may be merited. IE the destructor would unset the environment vairable.

Reply:
If a lot of tests are written using env-vars, definitely worth it. Since this is the only test so far, I'm happy to leave it for now.